docs/basics/integrations/slack.md

```diff
@@ -2,28 +2,92 @@
 id: slack
 title: Slack Integration
 sidebar_label: Slack
-description: Link your Sauce Labs account with your Slack workspace to set up auto-notification of your test results.
+description: Link your Sauce Labs account with your Slack workspace set up auto-notification of your test results.
 keywords:
 - share-test-results
 - slack
 - how-to
 ---
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <p><span className="sauceDBlue">COMING SOON</span></p>
 
 The Sauce Labs Slack app allows you to send test result notifications to selected workspace channels.
 
-<!-- ## What You'll Need
+## What You'll Need
 
 * A Sauce Labs account (if you don't have one, start a [free trial](https://saucelabs.com/sign-up))
 * Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
-* A [Slack](https://slack.com/) workspace with Admin rights
+* A [Slack](https://slack.com/) workspace
 
-## Install the App
+## Install the Slack Plugin
 
-:::note Slack Admin Required
-You must have Admin rights to the workspace in which you are installing the app.
-::: -->
+In order to capture your Sauce Labs user account information for use in Slack notifications, you must create a link between your Sauce account and your Slack workspace.
+
+1. From your Sauce Labs account, navigate to the [Account Integrations](https://app.staging.saucelabs.net/integrations) page.
+1. Click the Slack **Install** button.
+1. Click **Allow** to give Sauce Labs permission to access your Slack workspace.
+1. Enter your Sauce Labs account details in the **Slack Integration** form:
+    * **Sauce Username**: The Username value with which you log into Sauce Labs.
+    * **Access Key**: The [Access Key](https://app.saucelabs.com/user-settings) associated with your Sauce Labs account.
+    * **Primary Data Center**: The Data Center associated with your Sauce Labs account.
+1. Click **Submit here**.
+
+:::note Multiple Data Centers
```

Comment:
If for different data centers our customers use different users, then it's true. In case if the saucelabs user is the same for multiple datacenters, it's enough to install Sauce Slack App once. What test case do you want to describe?